test/src/main/java/org/apache/accumulo/test/functional/CloneTestIT.java

```diff
@@ -121,6 +125,12 @@ public void testProps() throws Exception {
 
   }
 
+  private void checkTableState(String table, Connector c, TableState expected) throws TException {
```

Comment:
Purely pedantic, so I'm sure this is ignorable - consider naming this `assertTableState` because it actually performs the assertion and will fail if the assertion fails.

Reply:
done